provisioner/docker/docker-hadoop.sh

```diff
@@ -196,7 +196,9 @@ provision() {
         ) &
     done
     wait
-    cat .error_msg_* 2>/dev/null && exit 1
+    if [ -f .error_msg_* ]; then
```

Comment:
`test -f` takes only one argument. If there are multiple files matching `.error_msg_*`, this results in syntax error?

```
$ touch foo.1 foo.2
$ if [ -f foo.* ] ; then echo foo ; fi
-bash: [: foo.1: binary operator expected
```

Reply:
Thank you,  I forgot considering about that. I'll fix this with another way.

Reply:
I changed to use `find` to be able to pick up multiple files.

```sh
touch .error_msg_1
touch .error_msg_0
if [ `find . -maxdepth 1 -type f -name ".error_msg_*" | wc -l` -gt 0 ]; then echo "true"; fi

true
```